rafal_szyinski/pins.h

```diff
@@ -0,0 +1,32 @@
+#ifndef __PINS_H__
+#define __PINS_H__
+
+#include "Arduino.h"
+
+const int on_off_button = 4;
```

Comment:
I think it doesn't matter because we assign values and not a calculations so `const` and `constexpr` have same effect here

Reply:
It matters because the `constexp` is filled in compilation level. Let check this out:
> `constexpr` essentially states that a variable or function can be resolved at compile time, but is not guaranteed.

https://stackoverflow.com/questions/66176642/difference-between-constexpr-and-define

Reply:
I would prefer `constexpr` rather than 'const' but okay. I will aprove.